### PR TITLE
Allow multiple Run for schedules

### PR DESCRIPTION
**Pull request by pbouchez**

### CODE REVIEW

File: templates/bareos-dir/schedule/schedule.conf.j2 [hunk 1]
```diff
@@ -1,9 +1,13 @@
 # {{ ansible_managed }}
 Schedule {
   Name = "{{ item.name }}"
-  Run = {{ item.full }}
+{% for line in item.full %}
```
Comment (btravouillon) [inline templates/bareos-dir/schedule/schedule.conf.j2:5]: This looks like a breaking change. Is it backward compatible?

I'm not sure the for-loop will work when `{{ item.full }}` is a string.

Comment (pbouchez) [reply templates/bareos-dir/schedule/schedule.conf.j2:5]: Yes, it's a non-backward compatible breaking change (schedules vars need adaptation). It's needed to be able to use multiple "Run" lines in schedules configuration. For example, when you want to run a backup every 6h, there's no syntax for it : you need to specify multiple runs : 

```
Schedule {
  Name = "Schedule"
  Run = Full weekly at 01:35
  Run = Incremental daily at 00:05
  Run = Incremental daily at 06:05
  Run = Incremental daily at 12:05
  Run = Incremental daily at 18:05
}
```

This is not possible without this change, as only a unique `item.incr` is accepted.

The schedules vars syntax goes from : 

```
- name: Schedule
  full: "Full weekly at 01:35"
  incr: "Incremental daily at 00:05"
```

to : 

```
- name: Schedule
  full:
    - "Full weekly at 01:35"
  incr:
    - "Incremental daily at 00:05"
    (- "Incremental daily at 06:05")
    (- "Incremental daily at 12:05")
    (- "Incremental daily at 18:05")
```

Comment (btravouillon) [reply templates/bareos-dir/schedule/schedule.conf.j2:5]: Would this work?

```
{% if item.full is iterable %}
{% for line in item.full %}
  Run = {{ line }}
{% endfor %}
{% else %}
  Run = {{ item.full }}
{% endif %}
```

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Review by ahmam — APPROVED** [review]

**Pull request merged**